### DISCUSSION TITLE
NextAuth v5  GET /en/api/auth/signin 400  ("Bad request.")

### DISCUSSION THREAD

**Discussion by codexlin** [nextauthjs/next-auth#11532] [Help] [answered]
How to fix it ,pls help !!! This is repo [link](https://github.com/look4u-ovo/Talko) 
    "next": "latest",
    "next-auth": "5.0.0-beta.20",
    "next-intl": "^3.17.2",
![image](https://github.com/user-attachments/assets/574e7faf-057f-4c90-a283-c8e4f85c3d12)

**Comment by k-taro56** [comment, marked as answer]
incortect
https://github.com/look4u-ovo/Talko/blob/a66161b74adec0dacb78ea15a057ecf8c930e386/src/auth/index.ts#L4

**Reply by k-taro56** [reply]
https://github.com/look4u-ovo/Talko/pull/1

**Reply by codexlin** [reply]
Your PR I've tried this before and it still doesn't work

**Reply by codexlin** [reply]
Is this BASE_PATH written incorrectly ? how fix it

**Reply by codexlin** [reply]
Yestetday, I've tried move to src/app/[local]/auth/xxx/route.ts  -> app/auth/xxx/route.ts  , but it aslo got (400 BAD request)

**Reply by codexlin** [reply]
I have resovled it , tks bro😊
